dwave/cloud/client/base.py

```diff
@@ -731,7 +731,7 @@ def _fetch_solvers(self,
                     solver_id=name, filter=dynamic_fields, no_cache_=True)
 
                 # note: avg_load may not exist in solver data
-                if 'avg_load' in load:
+                if (avg_load := load.get('avg_load')) is not None:
                     solver.avg_load = load.avg_load
```

Comment:
It looks like you meant to do
```suggestion
                if (avg_load := load.get('avg_load')) is not None:
                    solver.avg_load = avg_load
```

Reply:
Copy/paste issue form below. I actually switched to `.get` because I haven't implemented `__contains__` on the model. Will fix.